digital_land/pipeline.py

```diff
@@ -389,12 +390,26 @@ def save_csv(self, lookups_path=None, entries=None):
             f, fieldnames=self.schema.fieldnames, extrasaction="ignore"
         )
         writer.writeheader()
+
+        entity_values = []
+        if os.path.exists(self.old_entity_path):
```

Comment:
This seems to be tripping up the test. I'd suggest either mocking it so the test thinks the file exists, or create the DictReader in a try .. except block and silently ignore a FileNotFoundError (but maybe re-raise any other sort)